fastlane/spec/action_metadata_spec.rb

```diff
@@ -3,6 +3,10 @@
 describe Fastlane::Action do
   Fastlane::ActionsList.all_actions do |action, name|
     describe name do
+      before do
+        allow(FastlaneCore::FastlaneFolder).to receive(:path).and_return("./fastlane")
```

Comment:
@nafu Right, this all boils down to that line of code.
After applying my changes, I've reaslized that I only mock `./fastlane` path, but I should also mock the `fastlane` path, so this PR definitely needs improvements in terms of rspecs.